src/main/java/core/basesyntax/Circle.java

```diff
@@ -0,0 +1,21 @@
+package core.basesyntax;
+
+public class Circle extends Figure {
+    private final int radius;
+
+    public Circle(int radius, String color) {
+        super(color);
+        this.radius = radius;
+    }
+
+    @Override
+    public double getArea() {
+        return ((double) Math.round(Math.PI * radius * radius * 10) / 10);
```

Comment:
Creating a figure, don't pass expressions in the constructor. For better code readability, consider creating a variable for the calculated area before returning it.